info.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "sOCUtilities",
+    "version": "1.0.0",
+    "fsrMinCompatibility": "7.4.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "SOC Utilities",
+    "dependencies": [],
```

Comment:
You might want to depend upon SFSP

 {
            "name": "sOARFramework",
            "type": "solutionpack",
            "label": "SOAR Framework",
            "minVersion": null
        }

Reply:
This has been added